src/algParams.md

```diff
@@ -1,108 +1,108 @@
-- id-MLKEM768-RSA2048-HMAC-SHA256
-  - OID: 2.16.840.1.114027.80.5.2.50
-  - Label: "`QSF-MLKEM768-RSAOAEP2048-HMACSHA256`"
-  - Key Derivation Function (KDF): HMAC-SHA256
+- id-MLKEM768-RSA2048-SHA3-256
+  - OID: 2.16.840.1.114027.80.5.2.62
+  - Label: "`QSF-MLKEM768-RSAOAEP2048-SHA3-256`"
```

Comment:
As above, all these labels don't align with the label convention from draft-irtf-cfrg-concrete-hybrid-kems.

I'm not confident that they won't change that convention.